applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -220,10 +260,8 @@ int enter_datamode(slm_datamode_handler_t handler)
 		return -EINVAL;
 	}
 
-	/* reset UART for data mode */
-	uart_rx_disable(uart_dev);
-	k_sleep(K_MSEC(10));
-	(void)uart_receive();
+	/* Enable UART for data mode */
+	(void)uart_receive_enable();
```

Comment:
Why the return value is not checked here?

Reply:
Return value check added.